orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/DeadMessageHandler.kt

```diff
@@ -40,7 +41,7 @@ import org.springframework.stereotype.Component
     }
     return when (message) {
       is TaskLevel -> CompleteTask(message, TERMINAL)
-      is StageLevel -> CompleteStage(message, TERMINAL)
+      is StageLevel -> AbortStage(message)
```

Comment:
Is this unrelated to FAILED_CONTINUE bubbling up?

Reply:
Because I needed to remove the status param from `CompleteStage` I needed something else that was an unambiguous "just stop this stage" operation.

Reply:
👍